README.md

```diff
@@ -1,7 +1,7 @@
 # Calendar
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_calendar/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_calendar/report/html_report/)
+- [DEMO LINK](https://<Mary-Kantsedal>.github.io/layout_calendar/)
```

Comment:
```suggestion
- [DEMO LINK](https://Mary-Kantsedal.github.io/layout_calendar/)
```

Reply:
fix it please
https://github.com/mate-academy/layout_calendar/pull/1791/files#r1098912330